include/vrt.h

```diff
@@ -61,6 +61,7 @@
  *	VRT_l_beresp_body() changed
  *	VRT_Format_Proxy() added	// transitional interface
  *	VRT_AllocStrandsWS() added
+ *	VRT_call() added
```

Comment:
This will need to be moved up to after 2020-03-16.

Reply:
yes, I stopped basing this branch for now in order to help reviewers. Substantial squashing will probably happen before any final merge, and I will keep this conversation open as a reminder